validate/validate_linux.go

```diff
@@ -94,7 +94,7 @@ func (v *Validator) CheckLinux() (errs error) {
 	for index := 0; index < len(v.spec.Linux.Devices); index++ {
 		device := v.spec.Linux.Devices[index]
 		if !deviceValid(device) {
-			errs = multierror.Append(errs, fmt.Errorf("device %v is invalid", device))
+			errs = multierror.Append(errs, specerror.NewError(specerror.DevicesAvailable, fmt.Errorf("device %v is invalid", device), rspec.Version))
```

Comment:
I think it is not 'DevicesAvailable' error, it is simply 'device is invalid'.
DevicesAvailable is done here: 'https://github.com/opencontainers/runtime-tools/blob/master/cmd/runtimetest/main.go#L437'

Reply:
BTW, the 'deviceValid' function should move to validate_linux.go.
It might be in another PR.